README.rst

```diff
@@ -87,9 +86,9 @@ Please see CHANGELOG for more history.
 Releasing
 ---------
 
-1. Update the version number in ``django_jinja_markdown/__about__.py``.
-2. Add an entry to the change log in the README file.
-3. Tag the commit where you changed the above with the version number: e.g. ``1.21``.
+1. Update the version number in ``pyproject.toml``.
+2. Add an entry to the CHANGELOG file.
+3. Tag the commit with the version number: e.g. ``1.21``.
 4. Push the commit and tag to the github repo.
 5. Create a new GitHub release, selecting the tag you just pushed to specify the commit. Hit Publish.
 6. Github will build and release the package to PyPI. Monitor the progress via the Actions tab.
```

Comment:
```suggestion
6. GitHub will build and release the package to PyPI. Monitor the progress via the Actions tab.
```

Reply:
ha, 3 different ways to write gIThUB there in those 3 lines. Funny.

Reply:
I have to admit I missed a few similar ones in other readmes too, guess I should pay more attention next time I'm in the neighborhood…:/